components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/OAuth2JWTTokenValidator.java

```diff
@@ -115,6 +126,11 @@ public boolean validateAccessToken(OAuth2TokenValidationMessageContext validatio
             LoggerUtils.triggerDiagnosticLogEvent(OAuthConstants.LogConstants.OAUTH_INBOUND_SERVICE, null,
                     OAuthConstants.LogConstants.FAILED, "System error occurred.", "validate-jwt-access-token", null);
             throw new IdentityOAuth2Exception("Error while validating Token.", e);
+        } catch (OrganizationManagementException e) {
+            LoggerUtils.triggerDiagnosticLogEvent(OAuthConstants.LogConstants.OAUTH_INBOUND_SERVICE, null,
+                    OAuthConstants.LogConstants.FAILED, "Error while retrieving the ancestor organization.",
```

Comment:
```suggestion
                    OAuthConstants.LogConstants.FAILED, "Error while retrieving the organization hierarchy.",
```

Reply:
Resolved in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2086/commits/d7e2b618cc92abd5d74d9f0aee4a339f78803efd